queries.sql

```diff
@@ -122,3 +122,61 @@ COUNT(a.owner_id)
 FROM animals a
 INNER JOIN owners o ON o.id = a.owner_id
 GROUP BY o.full_name;
+
+-- DAY 4
+-- Who was the last animal seen by William Tatcher?
+SELECT vets.name AS vet_name, animals.name AS Last_seen_animal, date_of_visit AS visit_date FROM visits 
+JOIN animals ON animals.id = visits.animals_id 
+JOIN vets ON vets.id = visits.vets_id 
+WHERE vets.id = 1 ORDER BY date_of_visit desc 
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT animals.name) FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Stephanie Mendez';
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT vets.name AS vet_name, species.name AS species FROM specializations 
+RIGHT JOIN vets ON vets.id = specializations.vets_id 
+LEFT JOIN species ON species.id = specializations.species_id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT vets.name AS vet_name, animals.name AS Last_seen_animal, date_of_visit AS visit_date FROM visits 
+JOIN animals ON animals.id = visits.animals_id 
+JOIN vets ON vets.id = visits.vets_id 
+WHERE vets_id = 3 AND  date_of_visit BETWEEN '2020-04-01' AND '2020-08-20';
+
+-- What animal has the most visits to vets?
+SELECT animals.name, COUNT (animals_id) AS visits FROM visits 
+JOIN animals ON animals.id = visits.animals_id 
+JOIN vets ON vets.id = visits.vets_id GROUP BY animals.name 
```

Comment:
- [ ] I think you don't need to join the vets table here so kindly remove this line and then you van add group by COUNT(animals.name) ✨🔥♻🚀